### PR TITLE
Python SDK doc revamp

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Updated documentation for the Python SDK.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 1]
```diff
@@ -3,9 +3,11 @@ title: "Python"
 description: Use RudderStack’s Python SDK to send events to various destinations.
 ---
 
-The **RudderStack Python SDK** lets you track your customer event data from your Python applications and send it to your specified destinations via RudderStack.
+RudderStack's Python SDK lets you track and send the events from your Python applications to the specified destinations.
 
-Check out the [**GitHub codebase**](https://github.com/rudderlabs/rudder-sdk-python) to get a more hands-on understanding of the SDK.
+<div class="infoBlock">
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:8]: can remove this infoblock for consistency purposes

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 2]
```diff
@@ -15,17 +17,15 @@ Check out the [**GitHub codebase**](https://github.com/rudderlabs/rudder-sdk-pyt
 
 ## SDK setup requirements
 
-To set up the RudderStack Python SDK, the following prerequisites must be met:
+1. Sign up to [RudderStack Cloud](https://app.rudderstack.com/signup).
+2. <Link to="/dashboard-guides/sources/#adding-a-source">Set up a Python source</Link> in your dashboard. You should be able to see a <strong>write key</strong> for this source.
 
-- You need to set up a [**RudderStack account**](https://app.rudderstack.com/signup).
-- Once signed up, <Link to="/dashboard-guides/sources/#adding-a-source">set up a Python source</Link> in the dashboard. You should be able to see a **Write Key** for this source, as shown below:
+You will also need a data plane URL. Refer to the <Link to="/dashboard-guides/overview/#data-plane-url">Dashboard Overview</Link> guide for more information on the data plane URL and where to find it.
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:23]: can move this after the image as write key reflects in the image.

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 3]
```diff
@@ -41,57 +41,62 @@ pip install rudder-sdk-python
 To initialize the RudderStack client, run the following code snippet:
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:41]:
```suggestion
To initialize the SDK, run the following code snippet:
```

same in the heading:
Initializing the RudderStack client -> Initializing the SDK

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 4]
```diff
@@ -129,21 +135,23 @@ rudder_analytics.page('userid', 'Documentation', 'Sample Doc', {
 
 The `page` method parameters are as described below:
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:136]:
```suggestion
The `page` method parameters are as shown:
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 5]
```diff
@@ -153,70 +161,83 @@ rudder_analytics.screen('userid', 'Settings', 'Brightness', {
 
 The `screen` method parameters are as described below:
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:162]:
```suggestion
The `screen` method parameters are as shown:
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 6]
```diff
@@ -153,70 +161,83 @@ rudder_analytics.screen('userid', 'Settings', 'Brightness', {
...
 
 ```python
-rudder_analytics.group('userid', 'groupid', {
+rudder_analytics.group('1hKOmRA4GRlm', '12', {
     'name': 'Company',
     'domain': 'IT'
 })
 ```
 
 The `group` method parameters are as follows:
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:189]:
```suggestion
The `group` method parameters are as shown:
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 7]
```diff
@@ -153,70 +161,83 @@ rudder_analytics.screen('userid', 'Settings', 'Brightness', {
...
-| `timestamp`    | Date     | Optional                                  | The timestamp of the message's arrival.                                                                                    |
-
-## Alias
+| **Field**      | **Type** |  **Description**     |
+| :------------- | :------- |  :---------------- |
+| `user_id` <br/> <span style="color: #4D4DFF;font-size:12px;">Required, if <strong>anonymous_id</strong> is absent.</span>     | String   | Unique identifier for a user in your database.      |
+| `anonymous_id`  <br/> <span style="color: #4D4DFF;font-size:12px;">Required, if <strong>user_id</strong> is absent.</span> | String   |  The SDK automatically sets this identifier in cases where there is no unique identifier for the user. |
+| `group_id`  <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span>    | String   | Unique identifier of the group in your database.    |
+| `traits`       | Object   | An optional dictionary of the group's traits like `name`or `email`.     |
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:196]:
```suggestion
| `traits`       | Object   | An optional dictionary of the group's traits like `name` or `email`.     |
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 8]
```diff
@@ -153,70 +161,83 @@ rudder_analytics.screen('userid', 'Settings', 'Brightness', {
...
 </div>
 
 A sample `alias` call is as shown:
 
 ```python
-rudder_analytics.alias('previousid', 'userid')
+rudder_analytics.alias('previous_id', 'user_id')
 ```
 
 The `alias` method parameters are as mentioned below:
```
Comment (ishachopra0401) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:216]:
```suggestion
The `alias` method parameters are as shown:
```

**Review by ishachopra0401 — APPROVED** [review]

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx (hunk 3 above)
Comment (ameypv-rudder) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:41]:
```suggestion
To initialize the SDK, run the following code snippet:
```

**Review by itsdebs — APPROVED** [review]

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:8]:
```suggestion
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 9]
```diff
@@ -3,9 +3,11 @@ title: "Python"
 description: Use RudderStack’s Python SDK to send events to various destinations.
 ---
 
-The **RudderStack Python SDK** lets you track your customer event data from your Python applications and send it to your specified destinations via RudderStack.
+RudderStack's Python SDK lets you track and send the events from your Python applications to the specified destinations.
 
-Check out the [**GitHub codebase**](https://github.com/rudderlabs/rudder-sdk-python) to get a more hands-on understanding of the SDK.
+<div class="infoBlock">
+Refer to the SDK's <a href="https://github.com/rudderlabs/rudder-sdk-python">GitHub codebase</a> for the implementation-specific details.
+</div>
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:10]:
```suggestion
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx [hunk 10]
```diff
@@ -15,17 +17,15 @@ Check out the [**GitHub codebase**](https://github.com/rudderlabs/rudder-sdk-pyt
...
+2. <Link to="/dashboard-guides/sources/#adding-a-source">Set up a Python source</Link> in your dashboard. You should be able to see a <strong>write key</strong> for this source.
 
-- You need to set up a [**RudderStack account**](https://app.rudderstack.com/signup).
-- Once signed up, <Link to="/dashboard-guides/sources/#adding-a-source">set up a Python source</Link> in the dashboard. You should be able to see a **Write Key** for this source, as shown below:
+You will also need a data plane URL. Refer to the <Link to="/dashboard-guides/overview/#data-plane-url">Dashboard Overview</Link> guide for more information on the data plane URL and where to find it.
 
-<img src="../../../assets/event-stream-sources/python-write-key-new.png" alt="Python source write key" />
-
-- You will also need a data plane URL. Refer to the <Link to="/resources/glossary/#data-plane-url">Glossary</Link> for more information on the data plane URL and where to find it.
+<img src="../../../assets/event-stream-sources/python-source-write-key.png" alt="Python source write key" />
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:25]:
```suggestion
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx (hunk 9 above)
Comment (ameypv-rudder) [inline docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:10]:
```suggestion
```

File: docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/sources/event-streams/sdks/rudderstack-python-sdk.mdx:21]: as the webapp is not yet updated with the latest snippet, removed the image temporarily - will add the updated image later.

**Pull request merged**